F3FChrono/chrono/Chrono.py

```diff
@@ -29,6 +29,7 @@ def __init__(self):
         self.inStart = False
         self.mode=chronoType.none
         self.status=chronoStatus.InWait
+        self.penalty=False
```

Comment:
I think penalty should be double value (100, 200, 1000, ... )

Reply:
Joel,
In this case, do you want a GUI to select penalty value ?
I can place it above penalty button.

Reply:
Yes, I think it should be a good idea. You could choose between a 100 points penalty or a 1000 points penalty. It should be possible to add several penalties (1 for hitting a person and 1 for crossing the safety line)

Reply:
I add an issue for this